mako_config/mobile-bert.config

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is a protobuff text format representation of a Mako
+# BenchmarkInfo message.
+#
+# For more information about text format protobuffs see: go/proto-text-format
+#
+# For more information about BenchmarkInfo and the fields it contains see
+# https://github.com/google/mako/blob/master/spec/proto/mako.proto
+#
+# This particular message is configured to be passed into a create_benchmark
+# command via the mako command line tool.
+#
+# The difference between this file and update_benchmark.config is that the
+# benchmark_key is not defined.
+#
+# For assistance using the command line tool see go/mako-help.
+#
+benchmark_key: "5538704950034432"
+benchmark_name: "MobileBert (Pixel 4) (Time Unit: ms)"
+project_name: "iree"
+
+# Only owners can write to the benchmark
+owner_list: "hanchung@google.com"
+owner_list: "buildkite-agent@iree-oss.iam.gserviceaccount.com"
+
+# Define the name and type for x-axis of run charts
+input_value_info: {
+  value_key: "t"
+  label: "time"
+  type: TIMESTAMP
+}
+
+# Three metrics, define the names for y-axis values of
+metric_info_list: {
+  value_key: "c"
```

Comment:
What is the significance of the letters selected here?

Reply:
They suggest to use one or two chars: https://github.com/google/mako/blob/master/spec/proto/mako.proto#L91

`c` stands for `cpu`, `g` stands for `gpu`, and `v` stands for `vmla`.

Reply:
hmm that doesn't scale very well :P what about vulkan and metal in this?
I'd prefer the overhead of ~3 characters (relative to all the rest of this proto junk it's insignificant) and something we have a chance in hell of ever decoding and keeping consistent. 'vlk' 'mtl' (both common abbreviations used in the ecosystem) 'vmla' 'cpu' etc seem better.

Reply:
SG and done.

(I still have to figure out how to update runs with new keys before land and update the config, so I will merge this later if others look good.)

Reply:
I successfully updated the config, existing runs and current scripts used in CI.